webpack.config.js

```diff
@@ -11,7 +11,7 @@ const isProd = process.env.NODE_ENV === 'production';
 const webpackConfig = {
   mode: isProd ? 'production' : 'development',
   devtool: 'eval-source-map',
-  target: 'electron-renderer',
+  target: 'web',
```

Comment:
Why the change away for electron as the target?

Reply:
The frontend code cannot access the electron module anymore, which turned the frontend more into a web app.